Source/Scene/Model.js

```diff
@@ -1543,6 +1682,10 @@ define([
             }
         }
 
+        if (usesExtension(model, 'WEB3D_quantized_attributes')) {
+            vs = modifyShaderForQuantizedAttributes(vs, id, model, context);
```

Comment:
> Instead of searching through all the meshes for each shader, is it cleaner/faster to process the glTF and add any Cesium-specific properties if needed for later during loading? For example, search for modelMaterialsCommon(gltfDefaults( in this file.

We could cache mesh ids to program ids as a preprocessing step if that is agreeable